File: Reservationpitch.Application/DTOs/UserDtos/Response/UserResponseDto.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Reservationpitch.Application.DTOs.UserDtos.Response
{
    public class UserResponseDto
    {
        private string Id { get; set; } = string.Empty;
        public string Name { get; set; } = string.Empty;
        public string Email { get; set; } = string.Empty;
        public string PhoneNumber { get; set; } = string.Empty;
    }
}

File: App/stadium_reservation/lib/models/Dtos/Auth/userresponsedto.dart
class Userresponsedto {
  String? id;
  String? name;
  String? email;
  String? phoneNumber;
  String? noneHashedPassword;
  String? imageUrl;

  Userresponsedto(
      {this.id,
      this.name,
      this.email,
      this.phoneNumber,
      this.noneHashedPassword,
      this.imageUrl});

  factory Userresponsedto.fromJson(Map<String, dynamic> json) {
    return Userresponsedto(
        id: json['id'],
        name: json['name'],
        email: json['email'],
        phoneNumber: json['phoneNumber'],
        noneHashedPassword: json['noneHashedPassword'],
        imageUrl: json['imageUrl']);
  }
}
